src/runtime/tests/capabilities-resolver-test.ts

```diff
@@ -52,94 +51,65 @@ describe('Capabilities Resolver New', () => {
   it('fails creating keys with no factories', Flags.withDefaultReferenceMode(async () => {
     const resolver = new CapabilitiesResolver({arcId: ArcId.newForTest('test')});
     // Verify storage keys for none of the capabilities cannot be created.
-    await assertThrowsAsync(async () => resolver.createStorageKey(
-        unspecified, entityType, handleId));
-    await assertThrowsAsync(async () => resolver.createStorageKey(
-        inMemory, entityType, handleId));
-    await assertThrowsAsync(async () => resolver.createStorageKey(
-        inMemoryWithTtls, entityType, handleId));
-    await assertThrowsAsync(async () => resolver.createStorageKey(
-        onDisk, entityType, handleId));
-    await assertThrowsAsync(async () => resolver.createStorageKey(
-        onDiskWithTtl, entityType, handleId));
+    await assertThrowsAsync(async () => resolver.createStorageKey(unspecified, entityType, handleId));
+    await assertThrowsAsync(async () => resolver.createStorageKey(inMemory, entityType, handleId));
+    await assertThrowsAsync(async () => resolver.createStorageKey(inMemoryWithTtls, entityType, handleId));
+    await assertThrowsAsync(async () => resolver.createStorageKey(onDisk, entityType, handleId));
+    await assertThrowsAsync(async () => resolver.createStorageKey(onDiskWithTtl, entityType, handleId));
   }));
 
   it('creates volatile keys', Flags.withDefaultReferenceMode(async () => {
     // Register volatile storage key factory.
     // Verify only volatile (in-memory, no ttl) storage key can be created.
-    VolatileStorageKey.register();
+    //VolatileStorageKey.register();
```

Comment:
sorry, i'm confused - without registering the `VolatileStorageKey`here, this test is same as the one above, but they behave differently...